integration-tests/tests/example-page.cy.ts

```diff
@@ -70,11 +71,11 @@ if (!Cypress.env('OPENSHIFT_CI') || Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC')) {
     });
 
     it('Verify the url', () => {
+      cy.get('[data-quickstart-id="qs-nav-home"]').click();
+      cy.get('[data-test="nav"]').contains('Plugin Example').click();
       cy.url().should('include', '/example');
```

Comment:
I would just have one test that verifies the example page instead of splitting into two. The way the tests are written they're not independent anyway.